website/cue/reference/remap/functions/is_json.cue

```diff
@@ -0,0 +1,42 @@
+package metadata
+
+remap: functions: is_json: {
+	category: "Type"
```

Comment:
I just realized this is marked as a "type" function. I can see why you did that, but JSON isn't a type in VRL.

Having said that, let's ship this as is because I can see how people might expect a function like this to be present in that list. We can always change this in the future.